responsive-web-design-piano/css/style04.css

```diff
@@ -0,0 +1,22 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+
+/* piano styles */
+#piano {
+    width: 992px;
+    height: 290px;
+}
+
+/* 
+    Give a `margin` of `80px auto`, to set a margin 80px for top and bottom,
```

Comment:
Give the piano a `margin` of `80px auto` to set the margin to
80 pixels on the top and bottom, and to 
center it horizontally.